src/main/java/core/basesyntax/IsoscelesTrapezoidSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoidSupplier {
+    private static final RandomGeometricValue randomSize = new RandomGeometricValue();
+    private static final ColorSupplier color = new ColorSupplier();
+
+    public Figure getRandomIsoscelesTrapezoid() {

```

Comment:
The method 'getRandomIsoscelesTrapezoid' should be renamed to 'get' to comply with the method name in the 'FigureSupplier' interface or class.